frame/election-provider-multi-phase/src/benchmarking.rs

```diff
@@ -167,19 +167,19 @@ benchmarks! {
 		assert!(<MultiPhase<T>>::snapshot().is_none());
 		assert!(<MultiPhase<T>>::current_phase().is_off());
 	}: {
-		<MultiPhase<T>>::on_initialize_open_unsigned(true, true, 1u32.into());
+		<MultiPhase<T>>::on_initialize_open_unsigned(true, true, 1u32.into()).unwrap();
 	} verify {
 		assert!(<MultiPhase<T>>::snapshot().is_some());
 		assert!(<MultiPhase<T>>::current_phase().is_unsigned());
 	}
 
 	on_initialize_open_unsigned_without_snapshot {
 		// need to assume signed phase was open before
-		<MultiPhase<T>>::on_initialize_open_signed();
+		<MultiPhase<T>>::on_initialize_open_signed().unwrap();
```

Comment:
on all benchmarks you can always use `?` and it will bubble up the error

Reply:
Only if the return type is `DispatchResult` or its variant which can be made `Into<&str>`. This error can't and honestly I don't see much value in adding it since the short error message that we can add to `Into<&str>` or a `map_err(|_|)` will be probably poorer than just letting `unwrap` print the `Debug` implementation (which is informative), or at best only as good as what `.unwrap()`.